server/app/serializers/_kontena_json_serializer.rb

```diff
@@ -0,0 +1,12 @@
+class KontenaJsonSerializer < JsonSerializer
+
+  def decorate
+    return nil unless object
+
+    if object.is_a?(Array)
+      to_arry
```

Comment:
`to_arry` is right method call: https://github.com/frodsan/json-serializer/blob/master/lib/json_serializer.rb#L43-L51

The purpose of overriding this method is that Mongoid model has `#to_a` method that parent class uses to recognise arrays.